.circleci/create_circleci_config.py

```diff
@@ -171,6 +171,7 @@ def to_dict(self):
                     "command": f"TESTS=$(circleci tests split  --split-by=timings {self.job_name}_test_list.txt) && echo $TESTS > splitted_tests.txt && echo $TESTS | tr ' ' '\n'" if self.parallelism else f"awk '{{printf \"%s \", $0}}' {self.job_name}_test_list.txt > splitted_tests.txt"
                     }
             },
+            {"run": {"name": "fetch hub objects before pytest", "command": "python3 utils/fetch_hub_objects_for_ci.py"}},
```

Comment:
Before running pytest, let's load some objects (here some datasets).

Otherwise, because we use `pytest -n 8`, each of the 8 processes will possibility call hub